web3/contract.py

```diff
@@ -698,17 +698,17 @@ def _none_addr(datatype, data):
 )
 
 
-class ConciseMethod:
+class ConciseMethod(object):
```

Comment:
This shouldn't be necessary, web3 is py3-only.

Reply:
Does this happen under the hood for py3? I always thought `class MyObj` was py2-preferred and `class MyObj(object)` was py3

Reply:
> I always thought class MyObj was py2-preferred and class MyObj(object) was py3

It is the reverse, actually. `class MyObj(object)` was how you got some Py3-style object features into Py2. It is [(nearly) a no-op in Py3](https://stackoverflow.com/questions/1238606/is-it-necessary-or-useful-to-inherit-from-pythons-object-in-python-3-x).

Reply:
Ah, you learn something new everyday. Or more accurately, unlearn a bad habit lol.

Reply:
P.S. There are 26 instances of this in `web3/`:
```bash
$ grep -r "(object)" web3/ | wc -l
26
```

Reply:
Yeah, not a surprise that some are leftover from our py2 support days. Do you want to do the honors of deleting them? :)

If not, I'll take care of it.